modules/nw-installing-external-dns-operator-cli.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-external-dns-operator-cli_{context}"]
+= Installing the External DNS Operator by using the CLI
+
+You can install the External DNS Operator by using the CLI.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* You are logged into the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `Namespace` object:
+
+.. Create a YAML file that defines the `Namespace` object:
++
+.Example `namespace.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: external-dns-operator
+----
+
+.. Create the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f namespace.yaml
+----
+
+. Create an `OperatorGroup` object:
+
+.. Create a YAML file that defines the `OperatorGroup` object:
++
+.Example `operatorgroup.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: external-dns-operator
+  namespace: external-dns-operator
+spec:
+  upgradeStrategy: Default
+  targetNamespaces:
+  - external-dns-operator
+----
+
+.. Create the `OperatorGroup` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f operatorgroup.yaml
+----
+
+. Create a `Subscription` object:
+
+.. Create a YAML file that defines the `Subscription` object:
++
+.Example `subscription.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: external-dns-operator
+  namespace: external-dns-operator
+spec:
+  channel: stable-v1
+  installPlanApproval: Automatic
+  name: external-dns-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f subscription.yaml
+----
+
+.Verification
+
+. Get the name of the install plan from the subscription:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator \
+    get subscription external-dns-operator \
+    --template='{{.status.installplan.name}}{{"\n"}}'
+----
+
+. Check the status of the install plan:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator \
+    get ip <install_plan_name> \
+    --template='{{.status.phase}}{{"\n"}}'
+----
++
+.Required output
+[source,terminal]
+----
+Complete
+----
+
+. Check the status of the `external-dns-operator` pod is running:
```

Comment:
```suggestion
. Verify that the status of the `external-dns-operator` pod is `Running` by running the following command:
```
- Same comment about using "verify" instead of "check"
- Same comment using "by running the following command"
- The status of the pod is shown as "Running" with capital R.
- A pod status should be in back ticks.

Reply:
TY